modules/ipi-install-out-of-band-management.adoc

```diff
@@ -6,8 +6,18 @@
 [id="out-of-band-management_{context}"]
 = Out-of-band management
 
-Nodes will typically have an additional NIC used by the baseboard management controllers (BMCs). These BMCs must be accessible from the provisioner node.
+Nodes typically have an additional NIC used by the baseboard management controllers (BMCs). These BMCs must be accessible from the provisioner node.
 
 Each node must be accessible via out-of-band management. When using an out-of-band management network, the provisioner node requires access to the out-of-band management network for a successful {product-title} installation.
 
-The out-of-band management setup is out of scope for this document. We recommend setting up a separate management network for out-of-band management. However, using the provisioning network or the baremetal network are valid options.
+The out-of-band management setup is out of scope for this document. You can use a separate management network for out-of-band management. However, using the provisioning network or the baremetal network are also valid options.
+
+[NOTE]
+====
+You can use a maximum of two network interfaces on the bootstrap VM. If you configure a separate management network for out-of-band management, and you are using a provisioning network, the bootstrap VM requires routing access to the out-of-band network by using the bare-metal interface. In this scenario, the bootstrap VM can then access three networks:
+
+* the bare-metal network
+* the provisioning network
+* the management network routed through the bare-metal interface
```

Comment:
The [rest of the doc](https://docs.openshift.com/container-platform/4.12/installing/installing_bare_metal_ipi/ipi-install-prerequisites.html) never uses hyphenated bare-metal to refer to the baremetal network.

Reply:
It turns out the rest of the doc is wrong. I'll open a separate bug to fix it.